assets/stylesheets/style.scss

```diff
@@ -250,6 +250,7 @@
 
 // Just the tooltip WordPress Component styles is all we use for now.
 @import "~@wordpress/base-styles/colors";
+@import "~@wordpress/base-styles/variables";
```

Comment:
This has to be manually added here so @wordpress components can find it. I would have thought this would be handled automatically by npm/webpack, but this is also how wc-admin does things.